app/layout.tsx

```diff
@@ -15,7 +19,12 @@ export default function RootLayout({
   return (
     <html lang="en">
       <ThemeRegistry>
-        <body>{children}</body>
+        <body>
+          <StaticHeader />
```

Comment:
There was dublicated navbar for the FAQ page I removed it. And navbar was always stay open I fixed it